src/tests/ftest/control/config_generate_output.py

```diff
@@ -0,0 +1,447 @@
+                device_name = scm_dev.split("/")[-1]
+                if device_name not in self.scm_namespace_set:
+                    errors.append(
+                        "Cannot find SCM device name {} in expected set {}"\
+                            .format(device_name, self.scm_namespace_set))
+
+            # Verify the bdev_list values are in the NVMe PCI address set.
+            bdev_list = engine["bdev_list"]
+            for pci_addr in bdev_list:
+                if pci_addr not in self.pci_address_set:
+                    errors.append(
+                        "Cannot find PCI address {} in expected set {}".format(
+                            pci_addr, self.pci_address_set))
+
+            # Verify fabric interface values are in the interface set.
+            fabric_iface = engine["fabric_iface"]
+            if fabric_iface not in self.interface_set:
+                errors.append(
+                    "Cannot find fabric interface {} in expected set {}".format(
+                        fabric_iface, self.interface_set))
+
+        self.check_errors(errors)
+
+    def test_access_points(self):
+        """Test --access-points.
+
+        1. Call dmg config generate --access-points=wolf-a. Should pass.
+        2. Call dmg config generate --access-points=wolf-a:12345. Should pass.
+        3. Call dmg config generate --access-points=wolf-a:abcd. Should fail
+        because the port number is invalid.
+
+        :avocado: tags=all,full_regression
+        :avocado: tags=hw,small
+        :avocado: tags=control,config_generate_entries,access_points
+        """
+        errors = []
+        dmg = self.get_dmg_command()
+
+        # One host.
+        expected_ap = "wolf-a"
+        generated_yaml = dmg.config_generate(access_points=expected_ap)
+        actual_ap = generated_yaml["access_points"][0].split(":")[0]
+
+        if expected_ap != actual_ap:
+            msg = "One host failed! Expected = {}; Actual = {}".format(
+                expected_ap, actual_ap)
+            errors.append(msg)
+
+        # Host + valid port number.
+        expected_ap_port = "wolf-a:12345"
+        generated_yaml = dmg.config_generate(access_points=expected_ap_port)
+        actual_ap_port = generated_yaml["access_points"][0]
+
+        if expected_ap_port != actual_ap_port:
+            msg = "Host + valid port number failed! Expected = {}; Actual = "\
+                  "{}".format(expected_ap_port, actual_ap_port)
+            errors.append(msg)
+
+        # Host + invalid port number. Command should fail.
+        expected_ap_port = "wolf-a:abcd"
+        try:
+            generated_yaml = dmg.config_generate(access_points=expected_ap_port)
+            errors.append(
+                "Host + invalid port number succeeded! {}".format(
+                    expected_ap_port))
+        except CommandFailure as err:
+            self.log.info(
+                "Expected error from invalid port number: {}".format(err))
```

Comment:
(pylint-logging-format-interpolation) Use lazy % formatting in logging functions